src/cli/cli.ts

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+import { Command } from 'commander';
+import {Board} from "../board/board"
+import chalk from "chalk";
+
+let program : Command;
+const CANNOT_READ_FILE_ERROR = "Can't read file; please check your -f argument point to a valid file."
```

Comment:
Having nice error messages is not an explicit requirement, you could have saved yourself the trouble :wink: